scripts/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+SOURCE="github.com/pivotal-gss/gpmt2/cmd/gpmt"
+TARGET="build/gpmt"
+
+echo "building binary at build/gpmt"
+
+go build -o "${TARGET}" "${SOURCE}"
```

Comment:
Its good to have the OS information ( i.e GOOS & GOARCH ) with build, since a single build might not work with all the OS. so specifying the OS that we are sure we support might help..

Have a look at this blog for all the GOOS

https://www.digitalocean.com/community/tutorials/how-to-build-go-executables-for-multiple-platforms-on-ubuntu-16-04

Reply:
True - the scripts should be developed more, along with testing scripts. This is just an initial project framework.